q2dataflow/core/signature_converter/util.py

```diff
@@ -28,16 +28,22 @@ def get_mystery_stew(desired_filters=None):
     from q2_mystery_stew.plugin_setup import create_plugin
     from q2_mystery_stew.generators import FILTERS
 
-    pm = sdk.PluginManager.reuse_existing()
-
-    if desired_filters is not None:
-        _check_filters(desired_filters, FILTERS)
-    else:
-        desired_filters = {filter_: True for filter_ in FILTERS}
-
-    test_plugin = create_plugin(**desired_filters)
-
-    pm.add_plugin(test_plugin)
+    try:
+        pm = sdk.PluginManager.reuse_existing()
+    except sdk.UninitializedPluginManagerError:
+        pm = sdk.PluginManager(add_plugins=False)
+
+    try:
+        pm.get_plugin(id='mystery_stew')
+    except KeyError:
```

Comment:
duh... of course, thanks :)

I was just so baffled how the pluginmanager was already instantiated (turned out to be this: https://github.com/qiime2/q2cli/issues/334)